Documentation/guides/cpp_cmake.rst

```diff
@@ -29,18 +29,25 @@ Preparation
 
 #. Base NuttX compilation changes
 
-    For this example, load the configuration 'stm32f4discovery:testlibcxx' for building
+    For this example, load the configuration 'stm32f411e-disco:nsh' for building
```

Comment:
@leducp I suggest keeping the stm32f4discovery because it is a common/reference board for people using NuttX. For example it was used in almost all video tutorials on NuttX Channel.

Reply:
out of curiosity should we create a new config to update or replace testlibcxx or using nsh as base config is fine?